From: pancake <pancake@nopcode.org>
Subject: add kfreebsd support
Origin: upstream, http://radare.org/cgi-bin/hg/radare2/rev/e85eafa8d75c
diff -r 2b66dcba68a4 configure
--- a/configure	Thu Apr 07 23:26:14 2011 +0200
+++ b/configure	Wed Apr 13 10:43:22 2011 +0200
@@ -303,7 +298,7 @@
 exit 1 ; fi
 printf "checking for dynamic library... "
 HAVE_LIB_DL=0
-for OS in linux syllable sunos darwin beos solaris ; do
+for OS in linux syllable sunos darwin beos solaris gnu/kfreebsd ; do
 if [ "${HOST_OS}" = "${OS}" ]; then
 	HAVE_LIB_DL=1
 	break;
@@ -370,6 +365,9 @@
 if [ "$HOST_OS" = "linux" ]; then
 USEROSTYPE="gnulinux"
 else
+if [ "$HOST_OS" = "gnu/kfreebsd" ]; then
+USEROSTYPE="gnulinux"
+else
 if [ "$HOST_OS" = "netbsd" ]; then
 USEROSTYPE="gnulinux"
 else
@@ -380,7 +378,7 @@
 USEROSTYPE="gnulinux"
 else
 if [ "$HOST_OS" = "darwin" ]; then
-USEROSTYPE="darwin"; fi; fi; fi; fi; fi; fi
+USEROSTYPE="darwin"; fi; fi; fi; fi; fi; fi; fi
 SEDFLAGS=" -e '"
 COUNT=0
 for A in ${ENVWORDS} ; do
diff -r 2b66dcba68a4 libr/include/r_types.h
--- a/libr/include/r_types.h	Thu Apr 07 23:26:14 2011 +0200
+++ b/libr/include/r_types.h	Wed Apr 13 10:43:22 2011 +0200
@@ -72,14 +72,11 @@
 #undef __UNIX__
 #undef __WINDOWS__
 
-#if defined(__FreeBSD__) || defined(__NetBSD__) || defined(__OpenBSD__)
+#if defined(__linux__) || defined(__APPLE__) || defined(__FreeBSD_kernel__) || \
+    defined(__FreeBSD__) || defined(__NetBSD__) || defined(__OpenBSD__)
   #define __BSD__ 1
   #define __UNIX__ 1
 #endif
-#if defined(__linux__) || defined(__APPLE__)
-  #define __UNIX__ 1
-  #undef __BSD__
-#endif
 #if __WIN32__ || __CYGWIN__ || MINGW32
   #define __addr_t_defined
   #include <windows.h>
