From: pancake <pancake@nopcode.org>
Subject: honor --without-debugger
Origin: upstream, http://radare.org/cgi-bin/hg/radare2/raw-diff/b5639dbf6fda/libr/debug/p/debug_native.c

--- a/libr/debug/p/debug_native.c	Mon Apr 04 14:39:42 2011 +0200
+++ b/libr/debug/p/debug_native.c	Thu Apr 07 23:15:58 2011 +0200
@@ -7,11 +7,11 @@
 #include <r_lib.h>
 #include <signal.h>
 
+#if DEBUGGER
 static int r_debug_native_continue(int pid, int tid, int sig);
 static int r_debug_native_reg_read(RDebug *dbg, int type, ut8 *buf, int size);
 static int r_debug_native_reg_write(int pid, int tid, int type, const ut8* buf, int size);
 
-#define DEBUGGER 1
 #define MAXBT 128
 
 #if __WINDOWS__
@@ -115,6 +115,10 @@
 #define DEBUGGER 0
 #endif // ARCH
 
+#endif /* IF DEBUGGER */
+
+
+/* begin of debugger code */
 #if DEBUGGER
 
 #if __APPLE__
