Description: Typos.
 As subject.
Author: David Martínez Moreno <ender@debian.org>
Forwarded: no
Last-Update: 2014-06-24

Index: radare2/libr/core/cmd_help.c
===================================================================
--- radare2.orig/libr/core/cmd_help.c
+++ radare2/libr/core/cmd_help.c
@@ -191,7 +191,7 @@ static int cmd_help(void *data, const ch
 		"|  ~mov[0]       get first column of lines matching 'mov'\n"
 		"|  ~mov:3[0]     get 1st column from the 4th line matching 'mov'\n"
 		"|Misc:\n"
-		"|  0             go to begining of file\n"
+		"|  0             go to beginning of file\n"
 		"|  0x33          go to 0x33'\n"
 		);
 		return 0;
